www/js/survey/enketo/infinite_scroll_filters.js

```diff
@@ -1,40 +0,0 @@
-'use strict';
-
-/*
- * The general structure of this code is that all the timeline information for
- * a particular day is retrieved from the Timeline factory and put into the scope.
- * For best performance, all data should be loaded into the in-memory timeline,
- * and in addition to writing to storage, the data should be written to memory.
- * All UI elements should only use $scope variables.
- */
-
-import angular from 'angular';
-
-angular.module('emission.survey.enketo.trip.infscrollfilters',[
-    'emission.survey.enketo.trip.button',
```

Comment:
@sebastianbarry Angular is no longer registering the `emission.survey.enketo.trip.button` module because this module, and the other `infinite_scroll_filters`, are no longer depending on it.

So when you try to `getAngularService('EnketoTripButtonService')`, Angular finds nothing.

Include `emission.survey.enketo.trip.button` in the dependency array of `diary.js` and I think your issue will go away.